package/MDAnalysis/lib/_cutil.pyx

```diff
@@ -49,6 +49,122 @@ ctypedef cset[int] intset
 ctypedef cmap[int, intset] intmap
 
 
+@cython.boundscheck(False)  # turn off bounds-checking for entire function
+@cython.wraparound(False)   # turn off negative index wrapping for entire func
+cdef void _inverse_unique_unsorted_contiguous_array_inplace(
+                                                np.intp_t[::1] full_arr,
+                                                np.intp_t[::1] unique_arr,
+                                                np.intp_t[::1] mask
+                                                ):
+    """Find the inverse of a unique 1D array to map onto a non-unique 1D array.
+
+    Cython driver function. Loops though the unique array and finds the indices
+    in the full array that correspond to the values in the unique array.
+
+    ie: np.array([1,2,3])[np.array(0,0,1,2,2)] == np.array([1,1,2,3,3])
+               unique_arr[mask]                ==       full_arr
+
+
+
+    Parameters
+    ----------
+    full_arr: numpy.ndarray
+        1D contiguous array of dtype ``numpy.int64``,
+        the non-unique values.
+    unique_arr: numpy.ndarray
+        1D contiguous array of dtype ``numpy.int64``,
+        the unique values of full_arr, can be unsorted.
+    mask: numpy.ndarray
+        1D contiguous array of dtype ``numpy.int64``,
+        inverse array to be changed inplace, the same length as full_arr.
+
+    Returns
+    -------
+
+    Example
+    -------
+    Find the inverse of an array::
+
+        cdef full_arr = np.array([1,2,3,4,1,2,3,4], dtype=np.intp)
+        cdef unique_arr = np.array([1,2,3,4], dtype=np.intp)
+        cdef mask = np.empty(full_arr.shape[0], dtype=np.intp)
+    	_inverse_unique_unsorted_contiguous_array_inplace(full_arr,
+                                                          unique_arr,
+                                                          mask)
+       assert full_arr == unique_arr[mask]
+
+    """
+
+
+    """
```

Comment:
```suggestion
```

I think these extra quotes are breaking the compilation and therefore the test suite :)